src/js/controller/controller.js

```diff
@@ -577,15 +589,20 @@ Object.assign(Controller.prototype, {
             _actionOnAttach = null;
             checkAutoStartCancelable.cancel();
 
-            _model.set('pauseReason', _getReason(meta));
+            const state = _getState();
```

Comment:
Use `_model.get('state')`. Calling `_getState()` also calls `_getAdState()` which we don't want.